examples/.NET/README.md

```diff
@@ -0,0 +1,23 @@
+# GC Forms API client example in .NET
+
+## Prerequisites
+
+- Download and install .NET version 8.0
+  - (with Microsoft installer) https://dotnet.microsoft.com/en-us/download
+  - (with Brew) `brew install --cask dotnet`
```

Comment:
I had to use this package to get it running.  With the `dotnet` cask, I was getting errors about no SDKs installed:
```
brew install --cask dotnet-sdk
```

Reply:
Will update it! 
I could not really test because I already had dotnet installed on my machine even before working on this 😅